ioc_rix_sp1k1_calc/__main__.py

```diff
@@ -1,17 +1,35 @@
 import textwrap
+from os import environ
+from pathlib import Path
+from shutil import copy2
 
 import caproto.server
 
 from ioc_rix_sp1k1_calc import Ioc_rix_sp1k1_calc
 
 
+def install_archive(pv_names):
+    ioc_name = "ioc-rix-sp1k1-calc"
+    archive_file = ioc_name + '.archive'
+    data_dir = environ['IOC_DATA']
+    if not data_dir:
+        data_dir = '/reg/d/iocData'
+    archive_path = Path(data_dir, ioc_name, 'archive', archive_file)
+    local_archive = Path(Path(__file__).parent.parent, archive_file)
+    with open(local_archive, 'w') as f:
+        for pv in pv_names:
+            f.write(pv + "\t30 monitor\n")
+    copy2(local_archive, archive_path)
```

Comment:
It looks like if we run this multiple times, we're going to end up duplicating pv names each time.  (Each call writes `pv_names` to `local_archive`).  It would be better to create a new file (combining the packaged archive file with the rest of the PVs) and move it to the archive path.

Reply:
I think opening a file in `w` mode replaces the original file

Reply:
Ah my bad, that's right